test/utils/Constants.sol

```diff
@@ -8,5 +8,6 @@ abstract contract Constants {
     UD60x18 internal constant MAX_BROKER_FEE = UD60x18.wrap(0.1e18); // 10%
     uint40 internal constant MAX_UNIX_TIMESTAMP = 2_147_483_647; // 2^31 - 1
     uint128 internal constant MAX_UINT128 = type(uint128).max;
```

Comment:
I just noticed that this is not alphabetically ordered (should come before `MAX_UNIX_TIMESTAMP`).